packages/celotool/src/lib/generate_utils.ts

```diff
@@ -249,12 +250,12 @@ export const generateGenesisFromEnv = (enablePetersburg: boolean = true) => {
 }
 
 export const generateIstanbulExtraData = (validators: Validator[]) => {
-  const istanbulVanity = 32
+  const istanbulVanity = GENESIS_MSG_HASH
```

Comment:
Can we throw an error if istanbulVanity is not exactly equal to 64 characters?

Reply:
Good idea to be safe